code/pages/ExtensibleSearchPage.php

```diff
@@ -551,6 +551,13 @@ public function searchableTypes($default = null) {
 		return $listType;
 	}
 
+	public function SearchOverlay() {
+		$suggestions = $this->Suggestions()->filter(array(
+			'Approved' => 1,
+		))->sort('Frequency', 'DESC')->limit(5);
```

Comment:
I assume these have been loaded in to avoid making an AJAX query (using the autocomplete functionality), in which case should they be limited?